crates/node_binding/binding.d.ts

```diff
@@ -377,7 +377,7 @@ export interface JsLoaderContext {
    * @internal
    */
   diagnosticsExternal: ExternalObject<'Diagnostic[]'>
-  _moduleIdentifier: string
```

Comment:
Can we keep this field to the next minor/major version to avoid possible breaking changes?